tests/datasource/fluent/test_pandas_dbfs_datasource.py

```diff
@@ -0,0 +1,275 @@
+        DBFSDataConnector, csv_asset._data_connector
+    )
+    test_connection_error_message = f"""No file at base_directory path "{data_connector._base_directory.resolve()}" matched regular expressions pattern "{data_connector._batching_regex.pattern}" and/or glob_directive "**/*" for DataAsset "csv_asset"."""
+    return regex, test_connection_error_message
+
+
+@pytest.mark.integration
+def test_construct_pandas_dbfs_datasource(pandas_dbfs_datasource: PandasDBFSDatasource):
+    assert pandas_dbfs_datasource.name == "pandas_dbfs_datasource"
+
+
+@pytest.mark.integration
+def test_add_csv_asset_to_datasource(pandas_dbfs_datasource: PandasDBFSDatasource):
+    asset = pandas_dbfs_datasource.add_csv_asset(
+        name="csv_asset",
+        batching_regex=r"(.+)_(.+)_(\d{4})\.csv",
+    )
+    assert asset.name == "csv_asset"
+    assert asset.batching_regex.match("random string") is None
+    assert asset.batching_regex.match("alex_20200819_13D0.csv") is None
+    m1 = asset.batching_regex.match("alex_20200819_1300.csv")
+    assert m1 is not None
+
+
+@pytest.mark.integration
+def test_construct_csv_asset_directly():
+    # noinspection PyTypeChecker
+    asset = CSVAsset(
+        name="csv_asset",
+        batching_regex=r"(.+)_(.+)_(\d{4})\.csv",
+    )
+    assert asset.name == "csv_asset"
+    assert asset.batching_regex.match("random string") is None
+    assert asset.batching_regex.match("alex_20200819_13D0.csv") is None
+    m1 = asset.batching_regex.match("alex_20200819_1300.csv")
+    assert m1 is not None
+
+
+@pytest.mark.integration
+def test_csv_asset_with_regex_unnamed_parameters(
+    pandas_dbfs_datasource: PandasDBFSDatasource,
+):
+    asset = pandas_dbfs_datasource.add_csv_asset(
+        name="csv_asset",
+        batching_regex=r"(.+)_(.+)_(\d{4})\.csv",
+    )
+    options = asset.batch_request_options_template()
+    assert options == {
+        "path": None,
+        "batch_request_param_1": None,
+        "batch_request_param_2": None,
+        "batch_request_param_3": None,
+    }
+
+
+@pytest.mark.integration
+def test_csv_asset_with_regex_named_parameters(
+    pandas_dbfs_datasource: PandasDBFSDatasource,
+):
+    asset = pandas_dbfs_datasource.add_csv_asset(
+        name="csv_asset",
+        batching_regex=r"(?P<name>.+)_(?P<timestamp>.+)_(?P<price>\d{4})\.csv",
+    )
+    options = asset.batch_request_options_template()
+    assert options == {"path": None, "name": None, "timestamp": None, "price": None}
+
+
+@pytest.mark.integration
+def test_csv_asset_with_some_regex_named_parameters(
+    pandas_dbfs_datasource: PandasDBFSDatasource,
+):
+    asset = pandas_dbfs_datasource.add_csv_asset(
+        name="csv_asset",
+        batching_regex=r"(?P<name>.+)_(.+)_(?P<price>\d{4})\.csv",
+    )
+    options = asset.batch_request_options_template()
+    assert options == {
+        "path": None,
+        "name": None,
+        "batch_request_param_2": None,
+        "price": None,
+    }
```

Comment:
These 3 tests are not needed. `CSVAsset.batch_request_options` is already being tested elsewhere. All of the behavior is inherited.

Reply:
@NathanFarmer Good point.  Will be deleted in next commit.

Reply:
@NathanFarmer Done -- and corrected the names of these tests in the siblings of the `filesystem` ones.  Thanks.